apps/docs/src/components/code/ToggableLayout.tsx

```diff
@@ -59,7 +59,7 @@ export const ToggableLayout = ({ title, scope, code }: ToggableLayoutProps) => {
       {showPreview ? (
         <div
           className={clsx(
-            "p-md pt-lg min-h-400px mb-lg",
+            "p-md pt-lg min-h-100px mb-lg",
```

Comment:
if we use the (IMO much more suitable) `"expandable"` layout, we don't need to change this 🙃